Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -123,7 +123,7 @@ private function grabAttribsBeforeMethodToken(Tokens $tokens, $index)
      * @param Tokens $tokens Tokens collection
      * @param int    $index  token index
      *
-     * @return array array of grabbed attributes
+     * @return array<string, Token|null> array of grabbed attributes
```

Comment:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1487 and/or https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1512.

The 2nd one replaces the first one, but is breaking. I'm now suggesting take the non-breaking parts from the first, then apply them only to 1.11, then take the rest, and target the master.